### PR TITLE
Create es.json

### PR DESCRIPTION
**Pull request by EvertonMelo**
check this @j-a-m-l all your suggestions have here.

### CODE REVIEW

File: src/assets/i18n/es.json [hunk 1]
```diff
@@ -0,0 +1,245 @@
+{
+    "API": {
+      "BALANCE_TOO_LOW": "Saldo demasiado bajo para la transacción",
+      "BALANCE_TOO_LOW_DETAIL": "{{ token }} insuficiente en su cuenta! Los costes de transacción se: {{ totalAmount }} ({{ amount }} + {{ fee }} fee). Contudo, o saldo da sua conta é {{ balance }}!",
```
Comment (j-a-m-l) [inline src/assets/i18n/es.json]: Review the Portuguese here.

**Comment by j-a-m-l** [comment]
Please, review again the suggestions. If not, I will close this PR and create from scratch the translation on my own.

**Comment by EvertonMelo** [comment]
@j-a-m-l 

OK, 

so it's very sad but I'm giving up contributing to your project.

very unclear the process of contributions, very poor quality of feedback and you do not even read what you write after.

points to fixes it did not at other times ..

I think that's bad luck.

**Comment by j-a-m-l** [comment]
Sorry to hear that, but if you don't provide a good translation, I can't approve it.

**Comment by EvertonMelo** [comment]
@j-a-m-l 

it's not just that
but you have a very tight head.

You do not even mention anything you suggested.

soon he does not talk to illogical and subjective people.

**Comment by EvertonMelo** [comment]
The funny thing is before I send the translation, you have not done anything, so how can you be a good proofreader?

**Pull request closed without merging**